sqlmesh/core/console.py

```diff
@@ -286,6 +286,8 @@ def __init__(
 
         self.loading_status: t.Dict[uuid.UUID, Status] = {}
 
+        self.modified_threshold: int = 20
```

Comment:
No need for the type hint here:

```suggestion
        self.modified_threshold = 20
```

Also:
1) should we make this configurable?
2) if not, we can make it a class attribute, i.e.:

```python
class TerminalConsole(Console):
    INDIRECTLY_MODIFIED_DISPLAY_THRESHOLD = 20
```

Reply:
Ok, I'll make it a class attribute for now